src/test/java/com/powsybl/openloadflow/sa/OpenSecurityAnalysisTest.java

```diff
@@ -2024,6 +2027,47 @@ void testWithShuntAndGeneratorVoltageControls2() {
         assertEquals(133.557, result.getPostContingencyResults().get(0).getNetworkResult().getBusResult("BUS_2").getV(), LoadFlowAssert.DELTA_V);
     }
 
+    @Test
+    void testWithSharedVoltageRemoteControl() {
+        Network network = VoltageControlNetworkFactory.createWithSharedRemoteControl2GensOnSameBusAnd1Extra();
+        List<StateMonitor> monitors = List.of(new StateMonitor(ContingencyContext.all(), Set.of("l12", "l34"), Collections.emptySet(), Collections.emptySet()));
+        List<Contingency> contingencies = List.of(
+                new Contingency("g1", List.of(new GeneratorContingency("g1")))
+        );
+        LoadFlowParameters lfParameters = new LoadFlowParameters();
+        lfParameters.setUseReactiveLimits(true);
+        OpenLoadFlowParameters openLoadFlowParameters = OpenLoadFlowParameters.create(lfParameters);
+        openLoadFlowParameters.setReactivePowerRemoteControl(true)
+                .setMaxNewtonRaphsonIterations(30)
+                .setMaxRealisticVoltage(3)
+                .setStateVectorScalingMode(StateVectorScalingMode.LINE_SEARCH)
```

Comment:
functional: should not need to do this, this isn't a unit test any more. Likely linked to 1.6kV targetV